guides/introduction/getting-started.md

```diff
@@ -51,7 +51,7 @@ be able to run code in [Livebook](https://livebook.dev/) and work through
 Nerves tutorials from the comfort of your browser.
 
 [Underjord](https://www.youtube.com/c/Underjord) has put together a
-[fantastic video](https://github.com/nerves-livebook/nerves_livebook/raw/main/assets/video.jpg)](https://www.youtube.com/watch?v=-b5TPb_MwQE)
+[fantastic video](https://www.youtube.com/watch?v=-b5TPb_MwQE)
 to help walk-through the entire setup process.
```

Comment:
```suggestion
fantastic video to help walk-through the entire setup process.

[![Nerves Quickstart](https://github.com/nerves-livebook/nerves_livebook/raw/main/assets/video.jpg)](https://www.youtube.com/watch?v=-b5TPb_MwQE)
```

Another option would be to display the video thumbnail like we do in the [Nerves Livebook Firmware](https://github.com/nerves-livebook/nerves_livebook#nerves-livebook-firmware) README.md.

@fhunleth What do you prefer here?

Reply:
Thanks @onstop4!

@jjcarstens I'm going to do the easy thing and merge this, since it definitely fixes the typo. Then lets take a look.